package.json

```diff
@@ -2,77 +2,65 @@
-    "lint": "standard",
-    "build": "npm run build:js && npm run build:types",
-    "build:js": "ipjs build --tests --main && npm run build:copy",
-    "build:copy": "cp -a tsconfig.json src *.js dist/ && mkdir -p dist/test && cp test/*.js dist/test/",
-    "build:types": "npm run build:copy && cd dist && tsc --build",
-    "test:cjs": "npm run build:js && mocha dist/cjs/node-test/test-*.js && npm run test:cjs:browser",
-    "test:esm": "npm run build:js && mocha dist/esm/node-test/test-*.js && npm run test:esm:browser",
-    "test:node": "c8 --check-coverage --branches 100 --functions 100 --lines 100 mocha test/test-*.js",
-    "test:cjs:browser": "polendina --page --worker --serviceworker --cleanup dist/cjs/browser-test/test-*.js",
-    "test:esm:browser": "polendina --page --worker --serviceworker --cleanup dist/esm/browser-test/test-*.js",
-    "test:ts": "npm run build:types && npm run test --prefix test/ts-use",
-    "test": "npm run lint && npm run test:node && npm run test:esm && npm run test:ts",
-    "test:ci": "npm run lint && npm run test:node && npm run test:esm && npm run test:cjs && npm run test:ts",
-    "coverage": "c8 --reporter=html mocha test/test-*.js && npm_config_yes=true npx st -d coverage -p 8080"
-  },
-  "keywords": [
-    "ipfs",
-    "ipld",
-    "multiformats"
-  ],
   "author": "Rod <rod@vagg.org> (http://r.va.gg/)",
-  "license": "(Apache-2.0 AND MIT)",
-  "exports": {
-    "import": "./src/index.js"
-  },
-  "dependencies": {
-    "multiformats": "^9.5.4"
-  },
-  "devDependencies": {
-    "@types/chai": "^4.3.0",
-    "@types/mocha": "^9.0.0",
-    "c8": "^7.10.0",
-    "chai": "^4.3.4",
-    "chai-subset": "^1.6.0",
-    "ipjs": "^5.2.0",
-    "mocha": "^10.0.0",
-    "polendina": "~3.1.0",
-    "standard": "^17.0.0",
-    "typescript": "~4.8.2"
-  },
-  "standard": {
-    "ignore": [
-      "dist",
-      "test/ts-use/src/main.js"
-    ]
-  },
-  "directories": {
-    "test": "test"
-  },
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://github.com/ipld/js-dag-pb#readme",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ipld/js-dag-pb.git"
   },
   "bugs": {
     "url": "https://github.com/ipld/js-dag-pb/issues"
   },
-  "homepage": "https://github.com/ipld/js-dag-pb#readme",
+  "keywords": [
+    "ipfs",
+    "ipld",
+    "multiformats"
+  ],
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
+  },
+  "type": "module",
+  "types": "./dist/src/index.d.ts",
   "typesVersions": {
     "*": {
       "*": [
-        "types/*"
+        "*",
+        "dist/*",
+        "dist/src/*",
+        "dist/src/*/index"
       ],
-      "types/*": [
-        "types/*"
+      "src/*": [
+        "*",
+        "dist/*",
+        "dist/src/*",
+        "dist/src/*/index"
       ]
     }
   },
+  "files": [
+    "src",
+    "dist",
+    "!dist/test",
+    "!**/*.tsbuildinfo"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "import": "./src/index.js"
```

Comment:
so what is the purpose of `dist`? is it just meant to be the pieces that aren't checked in but are built at publish-time? i.e. the types?

Reply:
Might be useful to move it into the `types` folder the way we have for js-miltiformats. Or is `dist` the standard accross all the other IPFS repos?

Reply:
`dist` is standard across all other IPFS repos.

>  is it just meant to be the pieces that aren't checked in but are built at publish-time? i.e. the types?

Exactly, yes - types, minified bundles, source maps, all that stuff.